test/nl/surf/eduhub_validator_service/main_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns nl.surf.eduhub-validator-service.main-test
+  (:require [clojure.test :refer :all]
+            [nl.surf.eduhub-validator-service.main :as main]))
+
+(deftest test-validate-correct
+  (with-redefs [babashka.http-client/get (fn [& _] {:status 200 :body "mocked response"})]
```

Comment:
doe in de ns ook een require als je `babashka.http-client` direct aanspreekt.
kan je ook gelijk een alias gebruiken :-)

Ik zou hier zelf denk ik eerder `http-client/request` redefinen, maar dat is een beetje een smaak ding.